Core/include/Acts/Seeding/SeedFinderUtils.hpp

```diff
@@ -65,6 +66,23 @@ void transformCoordinates(std::vector<external_spacepoint_t*>& vec,
                           external_spacepoint_t& spM, bool bottom,
                           std::vector<LinCircle>& linCircleVec,
                           callable_t&& extractFunction);
+
+/// @brief Check the compatibility of spacepoint coordinates in xyz assuming the Bottom-Middle direction with the strip meassument details
+///
+/// @tparam external_spacepoint_t The external spacepoint type.
+/// @tparam sp_range_t Container type for the space point collections.
+///
+/// @param[in] m_config Seedfinder config containing the delegates to the strip measurement details.
+/// @param[in] sp Input space point used in the check.
+/// @param[in] spacepointPosition Spacepoint coordinates in xyz plane.
+/// @param[in] toleranceParam Parameter used to evaluate if spacepointPosition is inside the detector elements.
+/// @param[out] outputCoordinates The output vector to write to.
+/// @returns Boolean that says if spacepoint is compatible with being inside the detector element.
+template <typename external_spacepoint_t, typename sp_range_t>
+bool xyzCoordinateCheck(Acts::SeedfinderConfig<external_spacepoint_t> m_config,
```

Comment:
I think just by convention I'd call this argument `config`, since `m_*` implies it's a member of something.

Reply:
Also, and I didn't catch this before when `transformCoordinates` was added: for a global free function, this is a bit of a generic name. 

Doesn't have to be changed for this PR, but could you look into a follow up PR that changes the names of these free functions to something more specific?